db/schema.rb

```diff
@@ -686,6 +686,13 @@
     t.boolean "blocked", default: false, null: false
   end
 
+  create_table "visual_groups", charset: "utf8mb3", force: :cascade do |t|
```

Comment:
I don't think this should be part of this PR.

Reply:
Oh... that must have come in via merging master. I must have misunderstood what you were trying to say in the email thread... probably trying to save me the extra step. 

Removing from schema now.

Reply:
@mo-nathan When I run `bin/rails db:migrate" this keeps recreating the table in the schema, even though there is no migration in this branch that creates this table. 

I've forgotten how to resolve this situation! do you know offhand?

Reply:
It means the table exists in your database, but there's no corresponding migration.  The simplest thing to do is just copy db/schema.rb from master onto your branch and commit it.  To get `rails db:migrate` to work correctly, you can either reload the database or you can just go in to the database console and drop the table.  Then you should be able to run the migrations and schema.rb should not change.

Reply:
Got it. Doing that now. Thank you!